hs/src/Reach/AST/DLBase.hs

```diff
@@ -641,6 +641,7 @@ instance Pretty ApiInfo where
 data PrimVM -- Primitive Verification Mode
   = PV_Safe -- No static assertion, yes dynamic check
   | PV_Veri -- Yes static assertion, no dynamic check
+  | PV_None
```

Comment:
It doesn't seem like this is significant. Am I wrong?

Reply:
One of the issues I noticed with duoswap was that `tokenInfos` couldn't be erased because balance changes were impure. I don't want to make them `PV_Veri` because it would require users to make assertions. So this is useful for being considered pure and not requiring assertions. There are still assertions made about the balance before the operation, but not as part of `evalPrimOp`

Reply:
I see